src/js/components/List/List.js

```diff
@@ -93,6 +135,7 @@ const List = React.forwardRef(
     const theme = useContext(ThemeContext);
     const [active, setActive] = useState();
```

Comment:
Can we had a comment here explaining what 'active' means, since it has some interesting nuances in it when ordering?